cmd/saucectl/saucectl.go

```diff
@@ -70,7 +73,8 @@ func main() {
 	}
 }
 
-func setupLogging(verbose bool) {
+func setupLogging(verbose bool, noColor bool) {
+	color.NoColor = true
```

Comment:
Doesn't this need to depend on the value of the `noColor` arg?

Reply:
nice catch. fixed